contracts/ExtraFinalizeAgent.sol

```diff
@@ -38,17 +38,12 @@ contract ExtraFinalizeAgent is FinalizeAgent {
   uint public accountedTokenSales;
 
   function ExtraFinalizeAgent(CrowdsaleToken _token, Crowdsale _crowdsale, uint _bonusBasePoints, address _teamMultisig, uint _accountedTokenSales) {
+    require(address(_crowdsale) != 0 && address(_teamMultisig) != 0);
```

Comment:
Maybe two separate require()s?

Reply:
Changed to two separate require()s as requested.